JSON_serializer/main.py

```diff
@@ -0,0 +1,107 @@
+import requests
+import argparse
+import sys
+import os.path
+
+
+import serializer
+
+
+def create_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-f', dest='file_path', nargs='*', type=str,
+                        help='decode file of defined directory '
+                             '(which contains data in json format) from json to object')
+    parser.add_argument('-u', dest='url', nargs='*', required=False, type=str,
+                        help='decode data defined in URL from json to object ')
+    parser.add_argument('-j', dest='json_string', nargs='*', required=False, type=str,
+                        help='decode single quote string in json format from json to object')
+    parser.add_argument('unknown_data', nargs='*', type=str, help='call instruction')
+    return parser
+
+
+def is_valid_file(arg):
+    if not os.path.exists(arg):
+        print(f'The file {arg} does not exist. Check your input data.')
+        return False
+    else:
+        return True
+
+
+def is_url(address):
+    try:
+        url = requests.head(address)
+        return url.status_code == 200
+    except IOError:
+        print(f'The url {address} is not correct. Check your input data. ')
+        return False
+
+
+def file_argument(file_name):
+    file = open(file_name, 'r', encoding='utf-8')
+    string_from_file = ''
+    for line in file:
+        temp = line.strip()
+        string_from_file += temp
+    if string_from_file != '':
+        obj = serializer.check_format(string_from_file)
+        write_to_file(obj)
+        file.close()
+
+
+def write_to_file(decoded_string):
+    if len(str(decoded_string)) < 500:
+        print(decoded_string)
+    else:
+        out_file = open('output.txt', 'w', encoding='utf-8')
+        out_file.write(str(decoded_string))
+        print('Deserialized object is too long. It was written to output.txt in this directory.')
+        answer = input('Do you want to open this file in terminal? (y/n)')
+        if answer in ['y', 'yes', 'Y', 'YES']:
```

Comment:
`['y', 'yes', 'Y', 'YES']` this is constant block, isn't it? May be it would be good to move this constants file?